assets/section-image-banner.css

```
@@ -158,7 +158,25 @@
   left: auto;
 }
 
+.banner__media-half.animate--fixed:first-child>img {
+  width: 50%;
+}
+
+.banner__media-half.animate--fixed:nth-child(2)>img {
+  left: 50%;
+  width: 50%;
+}
```

This could probably be more general ? 
```suggestion
.banner__media-half.animate--fixed img {
  width: 50%;
}

.banner__media-half.animate--fixed:nth-child(2)>img {
  left: 50%;
}
```

---

Ah good catch 👍 Let's keep it as is, you're right.